<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Musik</class>
 <widget class="QGroupBox" name="Musik">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Music</string>
  </property>
  <property name="title">
   <string>Media</string>
  </property>
  <property name="flat">
   <bool>false</bool>
  </property>
  <property name="checkable">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="VlcWidgetVideo" name="video" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listTracks">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="AudioSpectrum" name="spectrum" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="chkRadio">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>continuous play</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/player/blue/radio.ico</normaloff>images/player/blue/radio.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="VlcWidgetSeek" name="seek" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdSave">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../tosec.qrc">
         <normaloff>:/icon/farious/blue/images/farious/blue/save.ico</normaloff>:/icon/farious/blue/images/farious/blue/save.ico</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="cmdEqualizer">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icon/player/blue/images/player/blue/equalizer.ico</normaloff>:/icon/player/blue/images/player/blue/equalizer.ico</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdPrev">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icon/player/blue/images/player/blue/backward.ico</normaloff>:/icon/player/blue/images/player/blue/backward.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdStop">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icon/player/blue/images/player/blue/stop.ico</normaloff>:/icon/player/blue/images/player/blue/stop.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdPause">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icon/player/blue/images/player/blue/pause.ico</normaloff>:/icon/player/blue/images/player/blue/pause.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdPlay">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icon/player/blue/images/player/blue/play.ico</normaloff>:/icon/player/blue/images/player/blue/play.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdNext">
       <property name="toolTip">
        <string>save actual file to disk</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icon/player/blue/images/player/blue/forward.ico</normaloff>:/icon/player/blue/images/player/blue/forward.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="VlcWidgetVolumeSlider" name="volume" native="true">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VlcWidgetVideo</class>
   <extends>QWidget</extends>
   <header>VLCQtWidgets/WidgetVideo.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VlcWidgetSeek</class>
   <extends>QWidget</extends>
   <header>VLCQtWidgets/WidgetSeek.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VlcWidgetVolumeSlider</class>
   <extends>QWidget</extends>
   <header>VLCQtWidgets/WidgetVolumeSlider.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AudioSpectrum</class>
   <extends>QWidget</extends>
   <header>src/widgets/AudioSpectrum/audiospectrum.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../tosec.qrc"/>
 </resources>
 <connections/>
</ui>
